frontend/app/src/store/monitor.ts

```diff
@@ -124,6 +125,12 @@ export const useMonitorStore = defineStore('monitor', () => {
     }
   });
 
+  const { ignoredAssets } = storeToRefs(useIgnoredAssetsStore());
+
+  watch(ignoredAssets, () => {
+    startPromise(fetchBlockchainBalances());
+  });
+
```

Comment:
So this will refetch all balances every time someone toggles the ignore/unignore for an asset right? what do we want to achieve here? maybe there is a more optimized way to achieve the result?

Reply:
So the ignored assets gets excluded from blockchain balances endpoint.
if we unignore, we need to refresh it.

I will optimize it, by only refresh particular chains, instead all

Reply:
@lukicenturi maybe then can we only make sure that it's called on unignore

Reply:
okay yeah we can do that, let me change it

Reply:
@kelsos updated